lib/Crypto/cryptomanager.h

```diff
@@ -313,6 +313,19 @@ class SAILFISH_CRYPTO_API CryptoManager : public QObject
     Q_DECLARE_FLAGS(Operations, Operation)
     Q_FLAG(Operations)
 
+    enum VerificationResult {
```

Comment:
How about just Verification instead of VerificationResult?  I just don't like shadowing the term Result.  Does anyone have strong feelings for or against?

Reply:
No strong feelings for or against but somehow Verification tastes better.

Reply:
+1

Reply:
Just "verification" seems to indicate a mode or type, when this is referring to a result. How about VerificationStatus? (I think that mentioned another time actually)

Reply:
VerificationStatus sounds good to me